helm/daemon/templates/genesis-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.deployment.runtimeConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: daemon-config
```

Comment:
if I understand correctly this is a file that is used by mina daemon. But also this is something mina-archive able to use(in another helm chart). So then it's not specific to daemon? if not, perhaps it should be called `runtime-config`?

Reply:
The mina-archive uses it in a very different way, to populate the genesis block in the database, and I think it can take different parameters, so idk if it should be explicitly shared / planned to be sharted with the archive like that.